src/EventStore.Core/Services/Transport/Grpc/PersistentSubscriptions.Read.cs

```diff
@@ -86,8 +87,13 @@ public partial class PersistentSubscriptions {
 				if (e == null) return null;
 				var position = Position.FromInt64(commitPosition ?? e.LogPosition, e.TransactionPosition);
 				return new ReadResp.Types.ReadEvent.Types.RecordedEvent {
-					Id = new UUID {
-						String = e.EventId.ToString()
+					Id = uuidOptionsCase switch {
```

Comment:
Do we want to default here if this is not set in the message?

Reply:
actually yeah